lib/client/doc.js

```diff
@@ -451,6 +477,14 @@ function pushActionCallback(inflight, isDuplicate, callback) {
   }
 }
 
+function combineCallbacks(callbacks) {
+  callbacks = callbacks.filter(Boolean);
```

Comment:
This is to filter out null/undefined items in `callbacks`?

Reply:
Yes. I can change to a `function` if you'd prefer. I tend to use this shorthand a lot personally (and I especially prefer it when arrow functions aren't available), but I can see why it can be confusing if you haven't come across it before. It's basically just abusing the fact that:

```javascript
Boolean(null) // false
Boolean(undefined) // false
Boolean(function() {}) // true
```

Could also just make some sort of util function called `truthy(arg) { return !!arg }` and use that for improved readability.